pkgs/tools/virtualization/google-compute-guest-configs/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, lib, util-linux, fetchFromGitHub, bash, coreutils, ethtool, gnugrep, makeWrapper }:
+let
+  version = "20211028.00";
+in stdenv.mkDerivation {
+  inherit version;
+  pname = "guest-configs";
+
+  src = fetchFromGitHub {
+    owner = "GoogleCloudPlatform";
+    repo = "guest-configs";
+    rev = version;
+    sha256 = "sha256-0SRu6p/DsHNNI20mkXJitt/Ee5S2ooiy5hNmD+ndecM=";
+  };
+
+  buildInputs = [ bash makeWrapper ];
```

Comment:
```suggestion
  nativeBuildInputs = [ makeWrapper ];

  buildInputs = [ bash ];
```

Reply:
We actually don't need bash here, not sure how it ended up here. Removed.